core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -687,7 +663,10 @@ public final class State implements Journaled {
     /**
      * Map from Alluxio path string to mount info.
      */
-    private final Map<String, MountInfo> mMountTable;
+    private final Map<String, MountInfo> mMountTable = new HashMap<>(10);
```

Comment:
nit: i believe hashmap works by power of twos so 10 here will actually allocate a hash function and space of size 16. I'm just looking at implementation of openjdk but i heard oracle jdk does the same. So maybe better off with 16 here.